### ISSUE TITLE
How can I know if the model is overfitting?

### ISSUE THREAD

**Issue by xuzhang5788** [kexinhuang12345/DeepPurpose#75]
From your demo and tutorials, you always set epoch=100, the learning rate is a constant, and you didn't show the comparison between the training losses and the validation losses. I saw somewhere in your codes for early stopping, but I don't know how to set it. Did you have a learning rate scheduling function? Thank you!

**Comment by kexinhuang12345** [comment]
Hi, yes, the demo and tutorials follow the suggestion by the DeepDTA paper. The early stopping is automatically set to avoid overfitting. Currently, no scheduler is defined. But it should super straightforward to add. For example, add after https://github.com/kexinhuang12345/DeepPurpose/blob/53002696640ebdee1bcc5c10ea8213e79e10ea81/DeepPurpose/DTI.py#L368 with https://pytorch.org/docs/stable/optim.html
